extensions/amp-story/amp-story.md

```diff
@@ -761,6 +761,125 @@ The `amp-story-cta-layer` allows mostly the same descendants as `amp-story-grid-
 
 For an updated list of supported children, be sure to take a look at the [amp-story-cta-layer-allowed-descendants](https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/validator-amp-story.protoascii) field in the validation rules.
 
+## Page attachments
+
+### `amp-story-page-attachment`
+
+<amp-img alt="AMP Story page attachment" layout="fixed" src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/amp-story-page-attachment.gif" width="240" height="480">
+  <noscript>
+    <img alt="AMP Story page attachment" src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/amp-story-page-attachment.gif" />
+  </noscript>
+</amp-img>
+
+Attach additional content to a story page!
+
+Story page attachments allow you to provide additional AMPHTML content to specific pages. This content can be revealed by users through a "swipe up" gesture, or a tap on the call to action element.
+A UI prompt to open the attachment will automatically be added at the bottom of every page that configured an attachment.
+
+The `<amp-story-page-attachment>` element must be the last child of `<amp-story-page>`, and must have the `layout="nodisplay"` attribute. The attachment AMPHTML content is expected to be provided inline in your AMP Story, within this `<amp-story-page-attachment>` tag.
+
+### Allowed content and components
+
+Story page attachments allow the same HTML elements as AMP Story, but also allow most other AMP components that usually can't be used in a story page, such as third party video players or social media embeds. This means you can create any experience: providing more text content, displaying various longer YouTube videos (with video controls!), tweets, Instagram posts...
```

Comment:
I think this should be less focused on expanding the types of content allowed or circumventing the validation of the page itself, but framed more from a content perspective, that it lets you provide additional information that may have been too long or optional that they didn't want to put it in the critical path of the story's navigation.

Reply:
Agreed, do you have product guidance on this? I was afraid that describing use cases like this might sound like an exhaustive list of what we're expecting, when it could be a space for publishers to get creative and create value in a way that works with the content they have. This is why I tried to focus the documentation on what they can put within an attachment, rather than use cases.

Should I phrase things around "additional information that may have been too long or optional"?

Reply:
Maybe @vinaymahag or @hongwei1990 have input on the baseline approach here; I'm also struggling to put it into words, and maybe @CrystalOnScript can help with that

Reply:
I took a stab at writing something more generic
Story page attachments allow most AMP components, even ones that are unavailable on an AMP story page. The attachment enables publishers to include content that does not fit on a page such as more details about the topic; or Twitter, Instagram, or YouTube embeds.

Reply:
Suggestion:
"Story page attachments allow the same HTML elements as AMP Story along with additional components listed below, such as third party video players or social media embeds. This mean you can add additional content that is too verbose or disallowed in an AMP Story page. "

Reply:
Thanks, @CrystalOnScript 's suggestion LGTM.